overlord/hookstate/ctlcmd/refresh.go

```diff
@@ -234,6 +260,27 @@ func getUpdateDetails(context *hookstate.Context) (*updateDetails, error) {
 	return &up, nil
 }
 
+func (c *refreshCommand) printTrackingInfo(context *hookstate.Context) error {
+	context.Lock()
+	defer context.Unlock()
+
+	st := context.State()
+	var snapst snapstate.SnapState
+	err := snapstate.Get(st, context.InstanceName(), &snapst)
+	if err != nil {
+		return fmt.Errorf("internal error: %v", err)
+	}
+
+	res, err := yaml.Marshal(map[string]string{"channel": snapst.TrackingChannel})
+	if err == nil {
+		c.print(string(res))
+	} else {
+		return fmt.Errorf("internal error: could not marshal tracking info: %v", err)
+	}
+
+	return nil
+}
```

Comment:
small tweak. It's simpler to do the usual `err != nil` 
```suggestion
	res, err := yaml.Marshal(map[string]string{"channel": snapst.TrackingChannel})
	if err != nil {
		return fmt.Errorf("internal error: could not marshal tracking info: %v", err)
	}

	c.print(string(res))
	return nil
}
```